app/models/grouping.rb

```diff
@@ -734,7 +734,7 @@ def get_total_test_script_marks
     total = 0
 
     #find the unique test scripts for this submission
-    test_script_ids = TestScriptResult.select(:test_script_id).where(:grouping_id => self.id)
+    test_script_ids = TestResult.select(:test_script_id).where(:grouping_id => self.id)
```

Comment:
Line is too long. [87/80]<br>Use the new Ruby 1.9 hash syntax.<br>Redundant `self` detected.